frontend/src/tests/test-server.js

```diff
@@ -0,0 +1,6 @@
+// src/mocks/server.js
+import { setupServer } from 'msw/node'
+import { handlers } from './handlers'
+
+// This configures a request mocking server with the given request handlers.
```

Comment:
@ZajoncM curious, are these comments auto-generated or added by you? In both cases, we can strip them as they do not convey any additional knowledge to the solution